src/main/java/core/basesyntax/service/FruitCounterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.ParsedLine;
+import java.util.List;
+import service.FruitCounter;
+import strategy.OperationStrategy;
+import strategy.implement.OperationStrategyImpl;
+
+public class FruitCounterImpl implements FruitCounter {
+    @Override
+    public boolean fruitCounter(List<ParsedLine> parsedLineList) {
+        OperationStrategy strategy = new OperationStrategyImpl();
```

Comment:
you have to make this a global variable and initialize it in the constructor, also, you have to pass the implementation as a parameter to the constructor